### DISCUSSION TITLE
Only install last specified dependency

### DISCUSSION THREAD

**Discussion by tildebox** [python-poetry/poetry#9760] [Q&A]
My pyproject.toml file looks like this.

```
[tool.poetry.dependencies]
psycopg = {version = "3.2.3", extras = ["c"]}

[tool.poetry.group.dev.dependencies]
psycopg = {version = "3.2.3", extras = ["binary"]}
```

For production environment I use:
`poetry install --only main`

For development environment I use:
`poetry install`

However, poetry installs psycopg[c] and psycopg[binary] in the development environment. How can I get it to install only psycopg[binary] in the development environment?

**Comment by radoering** [comment]
Maybe, `poetry install --only dev`?

**Reply by tildebox** [reply]
With `poetry install` the implicit main group and the dev group should be installed. Since I only need psycopg[binary] in the development environment, only that should be installed.
